### PR TITLE
Fixed button nvova

### PR DESCRIPTION
**Pull request by nvolodymyr**
add many styles for different buttons

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -35,3 +35,55 @@ body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
 .example-list.cdk-drop-list-dragging .example-box:not(.cdk-drag-placeholder) {
   transition: transform 250ms cubic-bezier(0, 0, 0.2, 1);
 }
+.primary-global-button{
+border: none;
```
Comment (nabermi2020) [inline src/styles.css]: general: bad alignment

File: src/app/component/general/eco-news/create-news/create-news.component.html [hunk 2]
```diff
@@ -9,8 +9,8 @@ <h2>Create news</h2>
     </div>
     <div class="info">
```
Comment (nabermi2020) [inline src/app/component/general/eco-news/create-news/create-news.component.html]: Please revert this file, it's out of the scope of your ticket

File: src/app/component/general/eco-news/filter-news/filter-news.component.css [hunk 3]
```diff
@@ -63,28 +63,29 @@
...
     opacity: 1;
+    background-image: url('../../../../../assets/img/icon/white-cross.png');
 }
 
 .closeNone{
     display: none;
 }
 
-.close::before,
+/* .close::before,
```
Comment (nabermi2020) [inline src/app/component/general/eco-news/filter-news/filter-news.component.css]: Please delete commented code

File: src/styles.css (hunk 1 above)
Comment (nvolodymyr) [reply src/styles.css]: fixed

File: src/app/component/general/eco-news/filter-news/filter-news.component.css (hunk 3 above)
Comment (nvolodymyr) [reply src/app/component/general/eco-news/filter-news/filter-news.component.css]: fixed

File: src/app/component/general/eco-news/create-news/create-news.component.html (hunk 2 above)
Comment (nvolodymyr) [reply src/app/component/general/eco-news/create-news/create-news.component.html]: fixed

File: src/app/component/general/homepage/homepage/homepage.component.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
       <div>
         <h1>{{ "homepage.header.caption" | translate }}</h1>
         <p>{{ "homepage.header.content" | translate }}</p>
-        <button class="button" (click)="startHabit()">{{ "homepage.general.button-start-habit" | translate }}</button>
+        <button class="button primary-global-button" (click)="startHabit()">{{ "homepage.general.button-start-habit" | translate }}</button>
```
Comment (KimAT25) [inline src/app/component/general/homepage/homepage/homepage.component.html]: move to 1 column class and (click)

File: src/app/component/general/homepage/stat-row/stat-row.component.html [hunk 5]
```diff
@@ -13,10 +13,10 @@ <h3>{{ "homepage.stats.bag-stat-before" | translate }}
           {{ "homepage.stats.bag-stat-after" | translate }}
         </h3>
         <p>{{ "homepage.stats.bag-question" | translate }}</p>
-        <button class="button">{{ "homepage.general.button-start-habit" | translate }}</button>
+        <button class="button primary-global-button">{{ "homepage.general.button-start-habit" | translate }}</button>
         <div class="location-row">
           <img [src]="locationImage" alt="location-image">
-          <a routerLink="/map">{{ "homepage.stats.bag-link" | translate }}</a>
+          <a class="tertiary-global-button" routerLink="/map">{{ "homepage.stats.bag-link" | translate }}</a>
```
Comment (KimAT25) [inline src/app/component/general/homepage/stat-row/stat-row.component.html]: alignment in 1 column

File: src/app/component/general/homepage/stat-row/stat-row.component.html [hunk 6]
```diff
@@ -29,10 +29,10 @@ <h3>{{ "homepage.stats.cup-stat-before" | translate }}
           {{ "homepage.stats.cup-stat-after" | translate }}
         </h3>
         <p>{{ "homepage.stats.bag-question" | translate }}</p>
-        <button class="button">{{ "homepage.general.button-start-habit" | translate }}</button>
+        <button class="button primary-global-button">{{ "homepage.general.button-start-habit" | translate }}</button>
         <div class="location-row">
           <img [src]="locationImage" alt="location-image">
-          <a routerLink="/map">{{ "homepage.stats.cup-link" | translate }}</a>
+          <a class="tertiary-global-button" routerLink="/map">{{ "homepage.stats.cup-link" | translate }}</a>
```
Comment (KimAT25) [inline src/app/component/general/homepage/stat-row/stat-row.component.html]: same as line 19

File: src/app/component/general/homepage/subscribe/subscribe.component.html [hunk 7]
```diff
@@ -16,7 +16,7 @@ <h2>{{ "homepage.subscription.caption" | translate }}</h2>
         [ngClass]="{'hidden': subscriptionError.length < 1, 'visible': !emailTouched || subscriptionError.length > 0}">
         {{ "homepage.subscription.subscription-error" | translate }}
       </p>
-      <button (click)="subscribe()">
+      <button class="primary-global-button" (click)="subscribe()">
```
Comment (KimAT25) [inline src/app/component/general/homepage/subscribe/subscribe.component.html]: alignment in 1 column

File: src/app/component/user/auth/sign-in/sign-in.component.html [hunk 8]
```diff
@@ -44,7 +44,7 @@ <h2>{{'user.auth.sign-in.fill-form' | translate}}</h2>
                <div class="forgot-wrapper">
                   <a (click)="openDialog()" class="forgot-password">{{'user.auth.sign-in.forgot-password' | translate}}</a>
                </div>
-               <button class="submit" [disabled]="!registerForm.valid" type="submit">
+               <button class="submit primary-global-button" [disabled]="!registerForm.valid" type="submit">
```
Comment (KimAT25) [inline src/app/component/user/auth/sign-in/sign-in.component.html]: alignment in 1 column line 47 and 48

File: src/app/component/general/homepage/homepage/homepage.component.html (hunk 4 above)
Comment (nvolodymyr) [reply src/app/component/general/homepage/homepage/homepage.component.html]: fixed

File: src/app/component/general/homepage/stat-row/stat-row.component.html (hunk 5 above)
Comment (nvolodymyr) [reply src/app/component/general/homepage/stat-row/stat-row.component.html]: +

File: src/app/component/general/homepage/stat-row/stat-row.component.html (hunk 6 above)
Comment (nvolodymyr) [reply src/app/component/general/homepage/stat-row/stat-row.component.html]: +

File: src/app/component/general/homepage/subscribe/subscribe.component.html (hunk 7 above)
Comment (nvolodymyr) [reply src/app/component/general/homepage/subscribe/subscribe.component.html]: +

File: src/app/component/user/auth/sign-in/sign-in.component.html (hunk 8 above)
Comment (nvolodymyr) [reply src/app/component/user/auth/sign-in/sign-in.component.html]: +

File: src/app/component/general/homepage/homepage/homepage.component.html (hunk 4 above)
Comment (nvolodymyr) [reply src/app/component/general/homepage/homepage/homepage.component.html]: +

File: src/app/component/general/homepage/stat-row/stat-row.component.html (hunk 5 above)
Comment (nvolodymyr) [reply src/app/component/general/homepage/stat-row/stat-row.component.html]: +

File: src/app/component/general/homepage/stat-row/stat-row.component.html (hunk 6 above)
Comment (nvolodymyr) [reply src/app/component/general/homepage/stat-row/stat-row.component.html]: +

File: src/app/component/general/homepage/subscribe/subscribe.component.html (hunk 7 above)
Comment (nvolodymyr) [reply src/app/component/general/homepage/subscribe/subscribe.component.html]: +

File: src/app/component/user/auth/sign-in/sign-in.component.html (hunk 8 above)
Comment (nvolodymyr) [reply src/app/component/user/auth/sign-in/sign-in.component.html]: +

File: src/app/component/general/eco-news/news-list/news-list.component.css [hunk 9]
```diff
@@ -34,32 +34,18 @@
 
 a {
   text-decoration: none;
+
```
Comment (nabermi2020) [inline src/app/component/general/eco-news/news-list/news-list.component.css]: Delete empty line

File: src/styles.css [hunk 10]
```diff
@@ -10,8 +10,12 @@ a {
   cursor: pointer;
 }
 
-html, body { height: 100%; }
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
```
Comment (nabermi2020) [inline src/styles.css]: general: add empty line after each class declaration

File: src/app/component/general/eco-news/create-news/create-news.component.html [hunk 11]
```diff
@@ -1,68 +1,5 @@
-<div class="container">
```
Comment (nabermi2020) [inline src/app/component/general/eco-news/create-news/create-news.component.html]: revert this file, it's out of the scope of your ticket and this functionality(news form) isn't done yet for implementation buttons styles

File: src/app/component/general/eco-news/news-list/news-list.component.css (hunk 9 above)
Comment (nvolodymyr) [reply src/app/component/general/eco-news/news-list/news-list.component.css]: fix

File: src/styles.css (hunk 10 above)
Comment (nvolodymyr) [reply src/styles.css]: fixed

File: src/app/component/general/eco-news/create-news/create-news.component.html (hunk 11 above)
Comment (nvolodymyr) [reply src/app/component/general/eco-news/create-news/create-news.component.html]: fixed, this task was 22 hours ago

File: src/app/component/general/eco-news/news-list/news-list.component.html [hunk 12]
```diff
@@ -2,7 +2,7 @@
   <div class="flex-container">
     <app-filter-news (gridOutput)="getFilterData($event)"></app-filter-news>
     <a [routerLink]="'/create-news'">
-      <div id="create-button">
+      <div id="create-button" class="secondary-global-button">
```
Comment (KimAT25) [inline src/app/component/general/eco-news/news-list/news-list.component.html]: put class and id in one column

**Pull request merged**